### PR TITLE
Change layer to network in MNIST example

### PR DESCRIPTION
**Pull request by craffel**

### CODE REVIEW

File: examples/mnist.py [hunk 1]
```diff
@@ -131,25 +131,26 @@ def build_custom_mlp(input_var=None, depth=2, width=800, drop_input=.2,
...
+                                        input_var=input_var)
     if drop_input:
-        layer = lasagne.layers.dropout(layer, p=drop_input)
+        network = lasagne.layers.dropout(network, p=drop_input)
     # Hidden layers and dropout:
     nonlin = lasagne.nonlinearities.rectify
     for _ in range(depth):
-        layer = lasagne.layers.DenseLayer(layer, width, nonlinearity=nonlin)
+        network = lasagne.layers.DenseLayer(
+                network, width, nonlinearity=nonlin)
```
Comment (craffel) [inline examples/mnist.py]: @f0k I just want you to notice how committed I am to consistency here!

Comment (f0k) [inline examples/mnist.py]: Very good! I think I'd have noticed and complained otherwise ;)

**Comment by f0k** [comment]
A good start, but the next function (`build_cnn()`) also calls everything `layer`, and excerpts of the code are in the tutorial (`docs/user/tutorial.rst`) and need to be adapted as well. Sorry if it looked like less effort than it is!

**Comment by craffel** [comment]
Done.

**Comment by f0k** [comment]
Great! Will merge tomorrow when Travis passed and nobody objects.

**Comment by benanne** [comment]
no objections from me :) looks good. and consistent!

**Comment by f0k** [comment]
Closing/Opening to get coveralls going over it, for the sake of the green checkmark.

**Pull request merged**